tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -253,6 +253,24 @@ def test_create_from_snapshot_with_env(self):
     self.assertEqual(req.env, ('VAR', 'VAL'))
 
 
+class TestInputFileCreation(TestBase, unittest.TestCase):
```

Comment:
```suggestion
class TestInputFileCreation(TestBase):
```
`TestBase` already subclasses `unittest.TestCase`!

Reply:
changed